data/events/scripts/player.lua

```diff
@@ -86,6 +86,25 @@ function Player:onLookInShop(itemType, count)
 end
 
 function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder, toCylinder)
+	-- Store Inbox
```

Comment:
> There for I don't understand why you wrote;

i said that because of your quote `And gl using a 1.1 datapack with 1.3 or even 1.2 without doing changes.`, which i never said all datapacks should be compatible with server.
> You should have to update your scripts, in no way should we care that a datapack has to be updated in order to implement new things.

as i said then datapack drop the status a "showcase" to user and become a dependency,(i mean scripts in datapack, not configuration files) nonetheless, how you ensure tfs new scripts is fully compatible with user custom script systems?